app/src/main/java/ru/vssemikoz/newsfeed/utils/Enum.java

```diff
@@ -0,0 +1,17 @@
+package ru.vssemikoz.newsfeed.utils;
+
+import ru.vssemikoz.newsfeed.models.Category;
+
+public class Enum {
```

Comment:
It's a bad idea to create a class with the reserved keyword name. Please, refer these links and then rewrite and move this code to your Category enum:

- https://www.baeldung.com/java-enum-iteration
- https://stackoverflow.com/questions/16149117/filling-a-list-with-all-enum-values-in-java